test/modules/TestResultCoordinator/Reports/DirectMethod/DirectMethodReport.cs

```diff
@@ -64,7 +64,15 @@ class DirectMethodReport : TestResultReportBase
         // MismatchFailure is when the there is a result in ActualStore but no result in ExpectedStore. This should never happen.
         public ulong MismatchFailure { get; }
 
-        public override string Title => $"DirectMethod Report ({this.ResultType}) for [{this.ExpectedSource}] and [{this.ActualSource}]";
+        public override string Title
+        {
+            get
+            {
+                return this.ReceiverSource.Match<string>(
```

Comment:
if you want to simplify, you can just call this.ReceiverSource.ToString().

Reply:
I think this prints out more nicely and is not so complicated

Reply:
Nevermind - changed it a bit